client/ayon_core/plugins/publish/extract_thumbnail.py

```diff
@@ -486,25 +486,38 @@ def _create_frame_from_video(self, video_file_path, output_dir):
         # Set video input attributes
         max_int = str(2147483647)
         video_data = get_ffprobe_data(video_file_path, logger=self.log)
-        # Use duration of the individual streams since it is returned with
-        # higher decimal precision than 'format.duration'. We need this
-        # more precise value for calculating the correct amount of frames
-        # for higher FPS ranges or decimal ranges, e.g. 29.97 FPS
-        duration = max(
-            float(stream.get("duration", 0))
-            for stream in video_data["streams"]
-            if stream.get("codec_type") == "video"
-        )
 
+        # Get duration or use a safe default (single frame)
+        duration = 0
+        for stream in video_data["streams"]:
+            if stream.get("codec_type") == "video":
+                stream_duration = float(stream.get("duration", 0))
+                if stream_duration > duration:
+                    duration = stream_duration
+
+        # For very short videos, just use the first frame
+        # Calculate seek position safely
+        seek_position = 0
+        # Only use timestamp calculation for videos longer than 0.1 seconds
+        if duration > 0.1:
+            seek_position = duration * self.duration_split
+
+        # Build command args
```

Comment:
The arguments building is confusing as hell...
```suggestion
        # Build command args
        cmd_args = []
        if seek_position > 0.0:
            cmd_args.extend(["--ss", str(seek_position)])
            
        cmd_args.extend(["-i", video_file_path])
        cmd_args.extend([
            "-analyzeduration", max_int,
            "-probesize", max_int,
            "-y",
            "-frames:v", "1",
        ])
```